javascript/builtins/webassembly/Exception.json

```diff
@@ -125,17 +125,85 @@
                   "version_added": "15.2"
                 },
                 "samsunginternet_android": {
-                  "version_added": false
+                  "version_added": "17.0"
                 },
                 "webview_android": {
-                  "version_added": false
+                  "version_added": "95"
                 }
               },
               "status": {
                 "experimental": false,
                 "standard_track": true,
                 "deprecated": false
               }
+            },
+            "options_parameter_traceStack": {
```

Comment:
Note, it occurs to be that this is not necessary, as it is the only option so far. I included it because it is is non-spec. But even if we decide it is spec I'd keep it - since then other options have something to compare again. Up to you though.